2008-04-21  Khem Raj  <kraj@mvista.com>

	* sysdeps/unix/sysv/linux/arm/ioperm.c: Don't include asm/page.h.

Index: ports/sysdeps/unix/sysv/linux/arm/ioperm.c
===================================================================
RCS file: /cvs/glibc/ports/sysdeps/unix/sysv/linux/arm/ioperm.c,v
retrieving revision 1.7
retrieving revision 1.8
diff -u -d -p -r1.7 -r1.8
--- ports/sysdeps/unix/sysv/linux/arm/ioperm.c	10 Jun 2005 11:12:09 -0000	1.7
+++ ports/sysdeps/unix/sysv/linux/arm/ioperm.c	21 Apr 2008 15:37:36 -0000	1.8
@@ -45,7 +45,6 @@
 #include <sys/mman.h>
 
 #include <linux/version.h>
-#include <asm/page.h>
 #include <sys/sysctl.h>
 
 #define PATH_ARM_SYSTYPE	"/etc/arm_systype"
